robot_smach_states/examples/manipulation/example_grasping.py

```diff
@@ -33,9 +36,10 @@
     pose = FrameStamped(frame=kdl.Frame(kdl.Rotation.RPY(0.0, 0.0, 0.0), kdl.Vector(args.x, args.y, args.z)),
                         frame_id="/map")
 
-    robot.ed.update_entity(id=entity_id, frame_stamped=pose)
+    shape = RightPrism([kdl.Vector(0, 0, 0), kdl.Vector(0, 0.05, 0), kdl.Vector(0.05, 0.05, 0), kdl.Vector(0.05, 0, 0)], -0.1, 0.1)
+    item = Entity("dummy_id", "dummy_type", pose.frame_id, pose.frame, shape, None, None, None)
 
-    item = ds.EdEntityDesignator(robot, id=entity_id)
+    item = ds.Designator(item)
```

Comment:
It might be use to mention here as documentation that the `resolve_type` of the `Designator` must be `Entity` for `Grab` to accept it.